WolvenKit.App/ViewModels/Shell/AppViewModel.cs

```diff
@@ -1714,7 +1718,7 @@ public void FinishedClosingModal()
     [NotifyCanExecuteChangedFor(nameof(SaveAllCommand))]
     private ObservableCollection<IDockElement> _dockedViews = new();
 
-    private double _uiScalePercentage => (double)SettingsManager.UiScale / 100.0;
+    private double UiScalePercentage => SettingsManager.UiScale / 100.0;
```

Comment:
Why this change?
Would prefer if we would keep the casing for privat class variables

Reply:
> Name '_uiScalePercentage' does not match rule 'non_field_members_should_be_pascal_case'. Suggested name is 'UiScalePercentage'.

Maybe needs fixing in the editorconfig

Reply:
Oh, missed that it is a property. nvm then